webapp/channels/src/components/more_direct_channels/list/list.tsx

```diff
@@ -105,13 +124,21 @@ const List = React.forwardRef((props: Props, ref?: React.Ref<MultiSelect<OptionV
             noteText={note}
             maxValues={MAX_SELECTABLE_VALUES}
             numRemainingText={
-                <FormattedMessage
-                    id='multiselect.numPeopleRemaining'
-                    defaultMessage='Use ↑↓ to browse, ↵ to select. You can add {num, number} more {num, plural, one {person} other {people}}. '
-                    values={{
-                        num: MAX_SELECTABLE_VALUES - props.values.length,
-                    }}
-                />
+                <div className='inline'>
+                    <FormattedMessage
+                        id='multiselect.numPeopleRemaining'
+                        defaultMessage='Use ↑↓ to browse, ↵ to select. You can add {num, number} more {num, plural, one {person} other {people}}. '
+                        values={{
+                            num: MAX_SELECTABLE_VALUES - props.values.length,
+                        }}
+                    />
+                    {'Please '}
+                    <span
+                        onClick={handleCreateChannel}
+                        className='link'
+                    >{'create a channel'}</span>
+                    {' to include more people.'}
```

Comment:
Use formatted Message for this, so it is translatable. You can use the chunks functionality to add the link (there are more examples in the code).